python/ray/_private/usage/usage_constants.py

```diff
@@ -1,4 +1,20 @@
+import os
+
 SCHEMA_VERSION = "0.1"
 
 # The key to store / obtain cluster metadata.
 CLUSTER_METADATA_KEY = b"CLUSTER_METADATA"
+
+# The usage collection server URL.
+# The environment variable is testing-purpose only.
+USAGE_REPORT_URL = os.getenv(
+    "RAY_USAGE_REPORT_URL", "https://usage-stats.ray.io/"
```

Comment:
Please add a test that setting this to some bogus / broken URL doesn't generate visible error messages (test_output.py).